content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -271,6 +271,13 @@ const Exhibition: FunctionComponent<Props> = ({
     />
   );
 
+  const exhibitionFormat =
+    exhibition.format?.title === 'Permanent exhibition'
+      ? 'Exhibition'
+      : exhibition.format?.title
+      ? exhibition.format.title
+      : 'Exhibition';
```

Comment:
_Very much_ a take it or leave it suggestion, your code already works very well so feel free to ignore, I just thought it might have a touch more readability, what do you think?
Also I haven't actually tested it, so who knows if it works. 

```suggestion
    !exhibition.format || exhibition.format?.title === 'Permanent exhibition'
      ? 'Exhibition'
      : exhibition.format.title;
```

Reply:
Tested and works perfectly. Great suggestion - thanks, Raph! In latest push.